src/linter/phpdoc_util.go

```diff
@@ -12,27 +12,61 @@ import (
 	"github.com/VKCOM/noverify/src/types"
 )
 
-type phpdocErrors struct {
-	phpdocLint []string
-	phpdocType []string
+type PHPDocPlace struct {
```

Comment:
Maybe we can move it to `phpdoc` package?
Along with related functions like PHPDocLine.

Also, Place should probably be called Location.

Reply:
Done.